tests/unit/conf/clusters/advanced.json

```diff
@@ -36,7 +64,12 @@
         "shard_level_metrics": [
           "IncomingBytes"
         ],
-        "shards": 1
+        "shards": 1,
+        "terraform_outputs": [
+          "user_arn",
```

Comment:
I should've checked the outputs in `tf_kinesis_streams` module in my first comment on this line. There is no `user_arn` output. We can output either `arn` or `stream_name`.
https://github.com/airbnb/streamalert/blob/03b7d160b76271aea96804f5a98128c90ab7688e/terraform/modules/tf_kinesis_streams/outputs.tf#L1-L7

Reply:
@chunyong-lin - in the same commit as the change you're referencing, I added this:

https://github.com/airbnb/streamalert/pull/1114/commits/b62b0ece2d2bcd4b447e9e12fe35c551cf90fc83#diff-517d65d9de13bb32b0329474e9b11627R13-R15